src/rules/react.ts

```diff
@@ -8,6 +8,8 @@ export default {
   'jsx-alignment': true,
   // Allows blacklisting of props in JSX with an optional explanatory message in the reported failure.
   'jsx-ban-props': false,
+  // When using a boolean attribute in JSX, you can set the attribute value to true or omit the value. // This rule will enforce one or the other to keep consistency in your code.
+  'jsx-boolean-value': true,
```

Comment:
We ended up changing the default to be `always`. The `tslint-react` maintainer preferred it over `never`.
[See PR](https://github.com/palantir/tslint-react/pull/70#issuecomment-285798353)